### ISSUE TITLE
Add alt text for images within a question

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#417]
> **Migrated from GitLab**
> **Original Author:** Dione Mentis (`@DioneMentis`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/59

`@yannis`  `@christos`  

## Context

As part of HHMI's accessibility requirements, all images should have alt-text. Questions authors (and anyone who has the permission to edit a question) needs a way to provide the alt text.

## Proposal

Add a field in the editor to provide alt-text

Have a look at this example question: https://hhmi.cloud68.co/question/2db4fdba-b5a5-4235-b336-fed9677cfc10

## Design

Follow the current design for adding captions. Add an addition field `Alternative text:` below caption. 

![Screenshot_2022-07-05_at_16.38.14](/uploads/df1ed91ee9f494fa0a31edc7d12b9554/Screenshot_2022-07-05_at_16.38.14.png)

## Implementation (if applicable)

[A description of the steps to implement the feature.]

## Alternative approaches (if applicable)

[Include any alternatives to meet this use case.]

## Open issues (if applicable)

[Links to and a discussion of related issues, if applicable.]
